### ISSUE TITLE
Style/EmptyCaseCondition wrongly autocorrects multiple values

### ISSUE THREAD

**Issue by maximeg** [bbatsov/rubocop#4283]
This is an issue about the `Style/EmptyCaseCondition` cop that wrongly autocorrects lines with multiples values.

--------

Given a file with :

```ruby
case
when my.foo?, my.bar?
  something
when my.baz?
  something_else
end
```

## Expected behavior

File should be modified to:

```ruby
if my.foo? || my.bar?
  something
elsif my.baz?
  something_else
end
```

## Actual behavior

File is modified to:

```ruby
if my.foo?, my.bar?
  something
elsif my.baz?
  something_else
end
```

Rubocop output is :

```
rubocop -D -a                
Inspecting 1 file
E

Offenses:

test.rb:1:1: C: [Corrected] Style/EmptyCaseCondition: Do not use empty case condition, instead use an if expression.
case
^^^^
test.rb:2:11: E: unexpected token tCOMMA
(Using Ruby 2.3 parser; configure using TargetRubyVersion parameter, under AllCops)
if my.foo?, my.bar?
          ^
test.rb:4:1: E: unexpected token kELSIF
(Using Ruby 2.3 parser; configure using TargetRubyVersion parameter, under AllCops)
elsif my.baz?
^^^^^
test.rb:6:1: E: unexpected token kEND
(Using Ruby 2.3 parser; configure using TargetRubyVersion parameter, under AllCops)
end
^^^
```

## RuboCop version

```
$ rubocop -V
0.48.1 (using Parser 2.4.0.0, running on ruby 2.3.3 x86_64-darwin16)
```

**Comment by Robbie-Smith** [comment]
I was able to reproduce the bug reported as well. I reproduced it using the same syntax and got the same results as above. 

```
rubicon -D -a
Inspecting 1 file
E

Offenses:

test.rb:1:1: C: [Corrected] Style/EmptyCaseCondition: Do not use empty case condition, instead use an if expression.
case
^^^^
test.rb:1:5: C: [Corrected] Style/TrailingWhitespace: Trailing whitespace detected.
case
    ^
test.rb:1:11: E: unexpected token tCOMMA
(Using Ruby 2.1 parser; configure using TargetRubyVersion parameter, under AllCops)
if my.foo?, my.bar?
          ^
test.rb:3:1: E: unexpected token kELSIF
(Using Ruby 2.1 parser; configure using TargetRubyVersion parameter, under AllCops)
elsif my.baz?
^^^^^
test.rb:5:1: E: unexpected token kEND
(Using Ruby 2.1 parser; configure using TargetRubyVersion parameter, under AllCops)
end
^^^

1 file inspected, 5 offenses detected, 2 offenses corrected
```

```
rubocop -v
0.48.1
```
